virtualenv/builders/legacy.py

```diff
@@ -114,8 +115,8 @@ def _get_base_python_info(self):
         # Get information from the base python that we need in order to create
         # a legacy virtual environment.
         return json.loads(
-            subprocess.check_output([
-                self.python,
+            check_output([
+                self._get_base_python_bin(),
```

Comment:
So, you're running a subprocess in ```_get_base_python_bin``` to get the argument for a subprocess call here. On Windows, the subprocess call overhead is significant - have you tested the performance of this against the current virtualenv?

I'm not saying I know of any *other* way to do it, but if there's a significant performance regression, we should make it clear. After all, virtualenv on Windows is already a bit slow (nothing like how it was before we used wheels, but that's the point - I'd hate to lose that gain).